libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -299,6 +299,21 @@ uint8_t AP_GPS::num_sensors(void) const
     }
 }
 
+bool AP_GPS::is_healthy(bool record_log, bool send_feedback) {
+    return is_healthy(0, record_log, send_feedback); //could be overloaded with defaults
+}
+
+bool AP_GPS::is_healthy(uint8_t instance, bool record_log, bool send_feedback) {
+    uint32_t tnow = AP_HAL::millis();
+    if (drivers[instance] != nullptr && state[instance].status != NO_GPS &&
+      tnow - timing[instance].last_message_time_ms < get_rate_ms(instance) &&
```

Comment:
@WickedShell So the 48 bytes here are not so relevant if only one method uses it...
You are right about the jitter i just though that healthy should litteraly mean is_healthy. On the issue #2879 a jitter buffer of some seconds would eat into the time for responding to a failing GPS. I will create the jitter buffer.
Should the jitter buffer be with a number of samples or time? Also should this buffer size be user configurable?